Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/polygon_soup_to_polygon_mesh.h

```diff
@@ -195,9 +197,12 @@ class Polygon_soup_to_polygon_mesh
   * @pre the input polygon soup describes a consistently oriented
   * polygon mesh.
   *
-  * @tparam PolygonMesh a model of `MutableFaceGraph` with an internal point property map
-  * @tparam Point a point type that has an operator `[]` to access coordinates
-  * @tparam Polygon a `std::vector<std::size_t>` containing the indices
+  * @tparam PolygonMesh a model of `MutableFaceGraph` with an internal point 
+  * property map
+  * @tparam PointRange a `RandomAccessContainer` with a value_type that has an 
```

Comment:
backticks

Reply:
You need to be more precise. You don't need to invent anything. A similar doc already exists [here](https://doc.cgal.org/latest/Polygon_mesh_processing/group__PMP__orientation__grp.html#gaadc5bac3072b11d2313e40043c729d0a).